src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -411,4 +414,27 @@ private function getSymfonyGhostAsSvg()
     }
+
+    private static function getTraceClassname($trace)
+    {
+        if (empty($trace['file'])) {
+            return null;
+        }
+
+        $vendorRoot = static::getVendorRoot();
+        $traceFileRelative = str_replace($vendorRoot, '', realpath($trace['file']));
+
+        if (strpos($traceFileRelative, '/vendor/') === 0) {
```

Comment:
It is a great idea reuse that. But currently the VarDumper is not a dependency of Debug component. Can I turn it a dependency?